### ISSUE TITLE
Type of MongoDB.Cursor requires 1 type argument

### ISSUE THREAD

**Issue by nutstick** [SierraSoftworks/Iridium#68]
I'm using typescript, but when I complied I got 
```
ERROR` in [at-loader] node_modules\iridium\dist\lib\Cursor.d.ts:19:13
        TS2314: Generic type 'Cursor<T>' requires 1 type argument(s).
ERROR in [at-loader] node_modules\iridium\dist\lib\Cursor.d.ts:27:78
        TS2314: Generic type 'Cursor<T>' requires 1 type argument(s).
```
I think MongoDB.Cursor need to provide a type argument. Any way to fix it?

**Comment by SPARTAN563** [comment]
Thanks for bringing this to my attention, looks like the underlying MongoDB client driver's type definitions have been updated to include type info. I'm updating Iridium now to take advantage of that and will have a release ready for you as soon as possible.

**Comment by SPARTAN563** [comment]
Should be fixed in v7.1.3 - feel free to re-open this issue if you run into any problems.

**Comment by nutstick** [comment]
Thanks for a quick solved!
